### ISSUE TITLE
Mercenaries, outfit, talking, bug

### ISSUE THREAD

**Issue by YALdysse** [Try/OpenGothic#684]
**Describe the bug**

**OpenGothic**: `v1.0.2878`
**OS:** `Linux Mint 21.3`

Steps to reproduce:
1. Take off the mercenary's clothes
1. Trying to talk to a mercenary and being rejected because of the wrong clothes
1. Put on a mercenary clothes
1. Trying to talk to a mercenary. Now the mercenary won't say any words.

**Save-game/Log-file/Screenshots/Video**
[save](https://drive.google.com/file/d/1PAwmLtxuh6f-ARtjQLm0Lg9dCGQkGXLh/view?usp=sharing)
[video](https://drive.google.com/file/d/1H6tPyNeF0cFDY4HuXH24Rg-292d6Bjcd/view?usp=sharing)

**Application log**
-

**Comment by Try** [comment]
Hi, @YALdysse !

I've tested your save. Even when replicating video, such as:
* show up as a Pirate
* change the closes to Mercenary
* talk (quick end)
* change back to Pirate
* attempt to talk again (rejected, due to wrong armor)
* change back to Mercenary
* talk (expected to not reacting)
  * here, in my case all works just fine

Are there any mods in your setup? Seems something related to `Npc_SetRefuseTalk`.

**Comment by Try** [comment]
No, I mean in my case there is a reaction - dialogs pops-up. By expected, I mean "expected, after watching the video".

Sorry, for confusion :)
